pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -726,6 +726,9 @@ type EtcdStatefulSetSettings struct {
 	DiskSize     *resource.Quantity           `json:"diskSize,omitempty"`
 	Resources    *corev1.ResourceRequirements `json:"resources,omitempty"`
 	Tolerations  []corev1.Toleration          `json:"tolerations,omitempty"`
+	// SpaceQuota is the storage size limit for etcd. Maximum allowed value for SpaceQuota is 8GB
+	// More Info: https://etcd.io/docs/v3.5/op-guide/maintenance/#space-quota
+	SpaceQuota *int64 `json:"spaceQuota,omitempty"`
```

Comment:
This gives a maximum of "8GB" but that's not a valid format for the field. IMO this should be clarified by renaming the field to `SpaceQuotaBytes`, or we should even think about making this a Megabyte field in the first place since you end up with really huge numbers if this is in bytes.

Reply:
Updated 🦖